packages/suite/jest.config.js

```diff
@@ -68,12 +76,13 @@ module.exports = {
         'node_modules',
         '<rootDir>/src/utils/suite/dom',
         '<rootDir>/src/utils/wallet/promiseUtils',
+        '<rootDir>/libDev',
     ],
     transformIgnorePatterns: ['/node_modules/(?!d3-(.*)|internmap)/'],
     testMatch: ['**/*.test.(ts|tsx|js)'],
     transform: {
-        '(d3-|internmap).*\\.js$': 'ts-jest',
-        '\\.(ts|tsx)$': 'ts-jest',
+        '(d3-|internmap).*\\.js$': ['babel-jest', babelConfig],
+        '\\.(ts|tsx)$': ['babel-jest', babelConfig],
```

Comment:
lets create followup issue to replace ts-jest with babel-jest in all packages

Reply:
https://github.com/trezor/trezor-suite/issues/4981